docs/merge.md

```diff
@@ -36,13 +36,17 @@ It’s also possible to annotate the imported or merged ontology axioms with the
 
   * `--annotate-derived-from true`: annotates all axioms with the source's version IRI if it exists, else with the ontology IRI, using `prov:wasDerivedFrom`. If the axiom already has an annotation using this property (`prov:wasDerivedFrom`), the existing annotation will be kept and no new annotation will be added.
 
+```
```

Comment:
These fenced code blocks will disable the ROBOT integration tests, which only run on indented code blocks. Did you intend to disable these tests?